src/DotLiquid/StandardFilters.cs

```diff
@@ -587,7 +587,7 @@ public static string Append(string input, string @string)
         public static string Prepend(string input, string @string)
         {
             return input == null
-                ? input
+                ? @string
                 : @string + input;
         }

```

Comment:
Condition becomes meaningless. If we do this change, we should simplify to 
```csharp
public static string Prepend(string input, string @string) => @string + input;
```

Reply:
what is the intended result of null + null? blank or null?

Reply:
It seems Ruby is blank in this case as well. While it seems above works, it might be clearer if we used string interpolation.

```c#
public static string Prepend(string input, string @string) => $"{@string}{input}";
```

Reply:
Well spotted! I've updated, and added a test.